### DISCUSSION TITLE
Welcome to yaml-spec Discussions!

### DISCUSSION THREAD

**Discussion by ingydotnet** [yaml/yaml-spec#81] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ioquatix** [comment]
Hello, I'm interested in contributing an RFC for indentation. Is this acceptable?

**Reply by ioquatix** [reply]
Here is my proposal: https://github.com/yaml/yaml-spec/pull/109

**Comment by ghost** [comment]
Hi all!  I would like to contribute some ideas and suggestions about YAML. I really like the yaml format and I want to help the community.

**Reply by ingydotnet** [reply]
The community chats here: https://matrix.to/#/#chat:yaml.io
